proposals/gc/Overview.md

```diff
@@ -211,11 +214,11 @@ Needs:
 * (mutually) recursive function types
 * down casts
 
-The down cast for the closure environment is necessary because expressing its static type faithfully would require first-class generics and existential types.
+The down cast for the closure environment is necessary because expressing a static type system would require first-class generics and existential types.
```

Comment:
Well, let's not get into the semantics of the word "express", but I've reworded a bit.